### PR TITLE
Updated Readme and admin info page

### PR DESCRIPTION
**Pull request by wpmonty**
#5 
Updated version
Updated tested WP version
Updated readme description and FAQ
Updated admin information page

### CODE REVIEW

File: functions/admin-pages.php [hunk 1]
```diff
@@ -83,19 +83,38 @@ function gnt_settings_page_content() {
  */
 function gnt_info_page_content() {
 	?><div class="wrap">
+
 		<h1><?php esc_attr_e( 'Information', 'get-notified' ); ?></h1>
+
+		<p>Get Notified sends you notifications when certain events happen on your site.</p>
```
Comment (kjbenk) [inline functions/admin-pages.php]: All text strings must be translated and escaped with `esc_html_e` or `esc_html__` depending on how you echo it.  Please address the other text strings in this file that are supposed to be html escaped but are attribute escaped instead.

File: functions/admin-pages.php [hunk 2]
```diff
@@ -83,19 +83,38 @@ function gnt_settings_page_content() {
...
+
+		<p>For now, the plugin simply sends a notification via email or a message to a Slack channel when a post changes status (i.e. publish, pending, draft, trash) your site.</p>
+
+		<h2><?php esc_attr_e( 'How to use Get Notified', 'get-notified' ); ?></h2>
+
+		<p>First, enable what you want to be notified about on the <a href="<?php echo get_admin_url() . 'admin.php?page=get-notified-hooks'; ?>">Hooks settings page</a>.</p>
```
Comment (kjbenk) [inline functions/admin-pages.php]: Needs an `esc_url`

File: functions/admin-pages.php [hunk 3]
```diff
@@ -83,19 +83,38 @@ function gnt_settings_page_content() {
...
+
+		<p>Then, select where you want to be notified on the <a href="<?php echo get_admin_url() . 'admin.php?page=get-notified-integrations'; ?>">Integrations settings page</a>.</p>
```
Comment (kjbenk) [inline functions/admin-pages.php]: Needs an esc_url

**Review by kjbenk — CHANGES_REQUESTED** [review]
Looks good, but you just need to add the proper translation functions as well was the escape functions :)

**Comment by wpmonty** [comment]
@kjbenk updated with text encoding and escaped url's. good to translate!

File: functions/admin-pages.php [hunk 4]
```diff
@@ -83,19 +83,50 @@ function gnt_settings_page_content() {
...
 function gnt_info_page_content() {
 	?><div class="wrap">
+
 		<h1><?php esc_attr_e( 'Information', 'get-notified' ); ?></h1>
-		<p>This is an open source project that I started to give developers the tools they need to create great notifications for a WordPress site. I welcome any help from developers who want to contribute :)</p>
-		<p>There are two main parts of the plugin: Hooks and Integrations. Hooks are basically the WordPress defined action hooks for any specific event. This can be when a post is published or when a theme is activated. Integrations are the tools used to notify people of these events, like email or slack. Developers can add new Hooks and/or Integrations to the plugin to perform the tasks they need.</p>
 
-		<a href="https://github.com/kjbenk/get-notified" target="_blank">Contribute - Read more</a> |
-		<a href="https://github.com/kjbenk/get-notified/issues" target="_blank">Report a Bug</a>
+		<p><?php esc_attr_e( 'Get Notified sends you notifications when certain events happen on your site.', 'get-notified' ); ?></p>
```
Comment (kjbenk) [inline functions/admin-pages.php]: Not a big deal, but please try to use the proper escaping function next time.  `esc_attr` is for an HTML attribute like `<p data-id="<?php echo esc_attr( $id ); ?>"></p>`, and the `esc_html` function is used for the actual HTML of the tag like`<p data-id="<?php echo esc_html( $content ); ?>"></p>`

**Review by kjbenk — CHANGES_REQUESTED** [review]
Fix the travis spacing and the escaping :)

https://travis-ci.org/kjbenk/get-notified/jobs/219000844#L305

**Comment by wpmonty** [comment]
@kjbenk all set!

**Review by kjbenk — APPROVED** [review]
Looks good to me 🐴  Make the updates for Travis to pass.

**Pull request merged**